From 1bb483859455903054df2322bf1925b9c02f600f Mon Sep 17 00:00:00 2001
From: Nicolas Fella <nicolas.fella@gmx.de>
Date: Tue, 8 Mar 2022 16:02:44 +0100
Subject: [PATCH] Fix missing KCoreAddons link

kgame includes kcoreaddons_export.h, so we need to link the target
---
 src/private/tests/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/private/tests/CMakeLists.txt b/src/private/tests/CMakeLists.txt
index 5033b40d..df8323e7 100644
--- a/src/private/tests/CMakeLists.txt
+++ b/src/private/tests/CMakeLists.txt
@@ -14,5 +14,5 @@ ecm_add_tests(
     kgamesvgdocumenttest.cpp
     kgamepropertytest.cpp
     NAME_PREFIX "libkdegamesprivate-"
-    LINK_LIBRARIES Qt5::Test KF5KDEGamesPrivate
+    LINK_LIBRARIES Qt5::Test KF5::CoreAddons KF5KDEGamesPrivate
 )
-- 
2.35.1

